Doc/library/codecs.rst

```diff
@@ -697,8 +697,8 @@ compatible with the Python codec registry.
 
    .. method:: writelines(list)
 
-      Writes the concatenated list of strings to the stream (possibly by reusing
-      the :meth:`write` method). The standard bytes-to-bytes codecs
+      Writes the concatenated iterable of strings to the stream. Infinite or
+      very large iterables are not supported. The standard bytes-to-bytes codecs
```

Comment:
```suggestion
      Writes the concatenated iterable of strings to the stream (possibly by reusing
      the :meth:`write` method). Infinite or
      very large iterables are not supported. The standard bytes-to-bytes codecs
```
No reason to remove this

Reply:
But wasn't it the case that we want to avoid specifying implementation details? And, while accurate, I'm not too sure what vague wording like 'possibly' brings, as (to me) it doesn't seem to help much

Reply:
We also want to minimize changes. I'd rather keep this change focused on the minimum needed to fix the original bug.

Reply:
Sure, that sounds good to me. Do you think the parenthesized phasing should be removed though? If so, I could open up a separate PR with just that change, otherwise I'll just leave it be.

Reply:
I don't think it's worth changing.